app/controllers/application_controller.rb

```diff
@@ -3,10 +3,17 @@
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
 
+  add_flash_types :success, :danger
```

Comment:
is this still necessary?

Reply:
i would imagine that it is still necessary cause i think devise might intervene and change the type and we then again convert it. But this is just a guess

Reply:
okay this is not from devise but from action pack gem and i would recommend using it to be allowed to use more than just alert and notice to be send.

Reply:
# Creates new flash types. You can pass as many types as you want to create
      # flash types other than the default <tt>alert</tt> and <tt>notice</tt> in
      # your controllers and views. For instance:
      #
      #   # in application_controller.rb
      #   class ApplicationController < ActionController::Base
      #     add_flash_types :warning
      #   end
      #
      #   # in your controller
      #   redirect_to user_path(@user), warning: "Incomplete profile"
      #
      #   # in your view
      #   <%= warning %>
      #
      # This method will automatically define a new method for each of the given
      # names, and it will be available in your views.